src/main/java/de/zalando/aruha/nakadi/service/EventStream.java

```diff
@@ -101,7 +99,7 @@ public void streamEvents() {
                     currentBatch = emptyBatch();
                     batchStartTime = currentTimeMillis();
 
-                    outputStream.write(BATCH_SEPARATOR.getBytes());
+                    outputStream.write(BATCH_SEPARATOR.getBytes(UTF8));
```

Comment:
What was the issue here? is `"\n".getBytes(UTF8)` different from `"\n".getBytes()`?